src/main/java/com/google/sps/servlets/MentorSignupServlet.java

```diff
@@ -41,6 +42,7 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     String username = request.getParameter("username");
```

Comment:
Same with this method - it's very long. Break this up into multiple helper functions - it'll be helpful for readability, and it'll be easier for all of you to figure out where things are actually happening in your code.

Reply:
Done!